cpp/src/arrow/acero/partition_util.h

```diff
@@ -57,7 +57,7 @@ class PartitionSort {
   /// output_pos_impl: [&out_arr] (int row_id, int pos) { out_arr[pos] = row_id; }
   ///
   /// After Execution
-  /// out_arr: [2, 5, 3, 5, 4, 7]
```

Comment:
LGTM, also would you mind set this a something like "values in out_arr indices"?

Reply:
ok